src/towncrier/build.py

```diff
@@ -145,6 +145,18 @@ def __main(
     base_directory, config = load_config_from_options(directory, config_file)
     to_err = draft
 
+    if project_version is None:
+        project_version = config.version
+        if project_version is None:
+            if not config.package:
+                raise UsageError(
+                    "'--version' is required since the the config file does "
```

Comment:
```suggestion
                raise UsageError(
                    "Passing '--version' is required since the config file does "
```

Reply:
Fixed the typo, but I think the "Passing" is unnecessary. The output looks like this for the usage error:
```
Usage: towncrier build [OPTIONS]
Try 'towncrier build --help' for help.

Error: '--version' is required since the config file does not contain 'version' or 'package'.
```

As an aside: it's pretty ugly that `build` uses a custom `--version` option, since it's overridden by the standard `--version` by click when accessed as the default command by the click `cli` group (e.g. `towncrier --version x.y` just returns `towncrier, version 22.12.1.dev0` rather than actually doing anything).

Reply:
> As an aside: it's pretty ugly that `build` uses a custom `--version` option, since it's overridden by the standard `--version` by click when accessed as the default command by the click `cli` group (e.g. `towncrier --version x.y` just returns `towncrier, version 22.12.1.dev0` rather than actually doing anything).

I think you can open an issue for that?

Reply:
Sure, done in #510